### ISSUE TITLE
⚠️ CAUTION - Site Owner Makes Disingenuous Claims About Site and Site Usage

### ISSUE THREAD

**Issue by burneraccount9876543211** [routman/publicnote.com#5]
Per the Readme (bolding my own),
1. Publicnote is a free online notepad. Enter any title to create a new note or to access any existing note. No account is required to access publicnote.
2. All notes are auto-saved to the cloud as you type. The check mark in the upper right corner indicates when a note has been successfully saved.
3. Notes stored on publicnote can be public or private, depending on the complexity of the title. 
4. Simple titles lead to highly **visible notes that can be edited by anyone who stumbles upon them**, while complex titles lead to private, anonymous notes that are securely encrypted. The higher the complexity of the title, the stronger the encryption.
5. Using common words as the title is for entertainment purposes only. You will find a variety of messages left by others. **If you find something you find offensive, just delete it.**

These statements are disingenuous, as there are several static pages on the site publicnote.com, that are uneditable, most notably the static page for israel, which reads "FREE PALESTINE"

Additionally, the source code in the repo does not match the code that is actually running on the site which also seems disingenuous. For example, the following is the compiled JS code from the site itself:
```
(function() {
    "use strict";
    var t = {
        830: function(t, e, o) {
            var r = o(144)
              , n = function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("div", {
                    attrs: {
                        id: "app"
                    }
                }, ["checkbox" === [t.hidden ? "password" : "text"] || "radio" === [t.hidden ? "password" : "text"] ? 0 : o("input", {
                    directives: [{
                        name: "model",
                        rawName: "v-model",
                        value: t.sot.title,
                        expression: "sot.title"
                    }],
                    attrs: {
                        placeholder: "title",
                        autocomplete: "off",
                        autocorrect: "off",
                        autocapitalize: "off",
                        spellcheck: "false",
                        type: [t.hidden ? "password" : "text"]
                    },
                    domProps: {
                        value: t.sot.title
                    },
                    on: {
                        keyup: function(e) {
                            return t.get()
                        },
                        input: function(e) {
                            e.target.composing || t.$set(t.sot, "title", e.target.value)
                        }
                    }
                }), o("div", {
                    directives: [{
                        name: "show",
                        rawName: "v-show",
                        value: !t.hidden,
                        expression: "!hidden"
                    }],
                    staticClass: "eye icon",
                    on: {
                        click: function(e) {
                            return t.hide()
                        }
                    }
                }), o("div", {
                    directives: [{
                        name: "show",
                        rawName: "v-show",
                        value: t.hidden,
                        expression: "hidden"
                    }],
                    staticClass: "closedeye icon",
                    on: {
                        click: function(e) {
                            return t.show()
                        }
                    }
                }), o("status", {
                    attrs: {
                        id: "status"
                    }
                }), "" == t.sot.title ? o("home", {
                    staticClass: "page"
                }) : "terms" == t.sot.title ? o("terms", {
                    staticClass: "page"
                }) : "suicide" == t.sot.title.toLowerCase() ? o("suicide", {
                    staticClass: "page"
                }) : "israel" == t.sot.title.toLowerCase() ? o("israel", {
                    staticClass: "page"
                }) : "about" == t.sot.title ? o("about", {
                    staticClass: "page"
                }) : o("textarea", {
                    directives: [{
                        name: "model",
                        rawName: "v-model",
                        value: t.sot.note,
                        expression: "sot.note"
                    }],
                    class: {
                        blur: t.sot.loading
                    },
                    attrs: {
                        placeholder: "type note here",
                        disabled: 1 == t.sot.loading
                    },
                    domProps: {
                        value: t.sot.note
                    },
                    on: {
                        keyup: function(e) {
                            return t.save()
                        },
                        input: function(e) {
                            e.target.composing || t.$set(t.sot, "note", e.target.value)
                        }
                    }
                })], 1)
            }
              , a = []
              , s = function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("div", [o("logo"), o("br"), t._v(" Enter any title above to access a note. No account required."), o("br"), o("br"), t._v(" Every note is public. You are free to modify any note you find."), o("br"), o("br"), t._v(" Every note is encrypted. You must know the title to see the note."), o("br"), o("br"), t._v(" Do not use publicnote to store sensitive information."), o("br"), o("br"), o("div", {
                    staticClass: "nav link",
                    on: {
                        click: function(e) {
                            t.sot.title = "about"
                        }
                    }
                }, [t._v("about")]), o("br")], 1)
            }
              , i = []
              , l = function() {
                var t = this
                  , e = t.$createElement;
                t._self._c;
                return t._m(0)
            }
              , c = [function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("div", [o("div", {
                    staticClass: "logo accent"
                }, [t._v("public")]), o("div", {
                    staticClass: "logo"
                }, [t._v("note ")])])
            }
            ]
              , d = {
                name: "logo"
            }
              , v = d
              , u = o(736)
              , h = (0,
            u.Z)(v, l, c, !1, null, "6f8dafb6", null)
              , p = h.exports
              , w = {
                name: "home",
                components: {
                    logo: p
                },
                data: function() {
                    return {
                        sot: this.$root.$data
                    }
                }
            }
              , _ = w
              , f = (0,
            u.Z)(_, s, i, !1, null, "6e6ae144", null)
              , g = f.exports
              , b = function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("div", [o("logo"), o("br"), t._v(" Terms of Service"), o("br"), o("br"), t._v(' By using the publicnote app or web app (“Service”), you are agreeing to be bound by the following terms and conditions (“Terms of Use”) set forth by publicnote.com (“We”). And when I say "We" I\'m talking about me and my dog.'), o("br"), o("br"), t._v(" I. We are not responsible for any information stored with the Service."), o("br"), o("br"), t._v(" II. We reserve the right to modify or terminate the Service for any reason, without notice at any time."), o("br"), o("br"), t._v(" III. We reserve the right to alter these terms at any time."), o("br"), o("br"), t._v(" IV. We reserve the right to refuse service to anyone for any reason at any time."), o("br"), o("br"), t._v(" V. Woof woof woof woof woof."), o("br"), o("br"), o("div", {
                    staticClass: "link",
                    on: {
                        click: function(e) {
                            t.sot.title = ""
                        }
                    }
                }, [t._v("home")]), o("br"), o("br")], 1)
            }
              , m = []
              , k = {
                name: "terms",
                components: {
                    logo: p
                },
                data: function() {
                    return {
                        sot: this.$root.$data
                    }
                }
            }
              , y = k
              , C = (0,
            u.Z)(y, b, m, !1, null, "0e46eff2", null)
              , S = C.exports
              , x = function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("div", [o("logo"), o("br"), t._v(" If you’re in emotional distress or suicidal crisis, know that someone is here to talk to you."), o("br"), o("br"), t._m(0), o("br"), t._v(" source: Google, 2022"), o("br"), o("br"), o("div", {
                    staticClass: "link",
                    on: {
                        click: function(e) {
                            t.sot.title = ""
                        }
                    }
                }, [t._v("home")]), o("br"), o("br")], 1)
            }
              , E = [function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("table", [o("thead", [o("tr", [o("th", [o("strong", [t._v("Country")])]), o("th", [o("strong", [t._v("Hotline organization")])]), o("th", [o("strong", [t._v("Website")])]), o("th", [o("strong", [t._v("Phone number")])])])]), o("tbody"), o("tbody", [o("tr", [o("td", [t._v("Argentina")]), o("td", [t._v("Centro de Asistencia al Suicida")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "http://www.asistenciaalsuicida.org.ar",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.asistenciaalsuicida.org")])]), o("td", [t._v("(011) 5275-1135")])]), o("tr", [o("td", [t._v("Australia")]), o("td", [t._v("Lifeline Australia")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.lifeline.org.au/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.lifeline.org")])]), o("td", [t._v("13 11 14")])]), o("tr", [o("td", [t._v("Austria")]), o("td", [t._v("TelefonSeelsorge Österreich")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.telefonseelsorge.at/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.telefonseelsorge.at")])]), o("td", [t._v("142")])]), o("tr", [o("td", [t._v("Belgium")]), o("td", [t._v("Centre de Prévention du Suicide")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.preventionsuicide.be/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.preventionsuicide.be")])]), o("td", [t._v("0800 32 123")])]), o("tr", [o("td", [t._v("Belgium")]), o("td", [t._v("Zelfmoord 1813")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.zelfmoord1813.be/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.zelfmoord1813.be")])]), o("td", [t._v("1813")])]), o("tr", [o("td", [t._v("Brazil")]), o("td", [t._v("Centro de Valorização da Vida")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.cvv.org.br/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.cvv.org")])]), o("td", [t._v("188")])]), o("tr", [o("td", [t._v("Canada")]), o("td", [t._v("Crisis Services Canada")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.crisisservicescanada.ca/fr/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.crisisservicescanada.ca/fr")])]), o("td", [t._v("833-456-4566")])]), o("tr", [o("td", [t._v("Chile")]), o("td", [t._v("Ministry of Health of Chile")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.hospitaldigital.gob.cl/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.hospitaldigital.gob")])]), o("td", [t._v("6003607777")])]), o("tr", [o("td", [t._v("China")]), o("td", [t._v("北京心理危机研究与干预中心")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "http://www.crisis.org.cn/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.crisis.org")])]), o("td", [t._v("800-810-1117")])]), o("tr", [o("td", [t._v("Costa Rica")]), o("td", [t._v("Colegio de Prof. en Psicología")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://psicologiacr.com/aqui-estoy/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("psicologiacr.com/aqui-estoy")])]), o("td", [t._v("2272-3774")])]), o("tr", [o("td", [t._v("France")]), o("td", [t._v("SOS Amitié")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.sos-amitie.org/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.sos-amitie.org")])]), o("td", [t._v("09 72 39 40 50")])]), o("tr", [o("td", [t._v("Germany")]), o("td", [t._v("TelefonSeelsorge Deutschland")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "http://www.telefonseelsorge.de",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.telefonseelsorge.de")])]), o("td", [t._v("0800 1110111")])]), o("tr", [o("td", [t._v("Hong Kong")]), o("td", [t._v("Suicide Prevention Services")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "http://www.sps.org.hk",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.sps.org")])]), o("td", [t._v("2382 0000")])]), o("tr", [o("td", [t._v("India")]), o("td", [t._v("iCall Helpline")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://icallhelpline.org/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("icallhelpline.org")])]), o("td", [t._v("9152987821")])]), o("tr", [o("td", [t._v("Ireland")]), o("td", [t._v("Samaritans Ireland")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.samaritans.org/how-we-can-help/contact-samaritan",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.samaritans.org/how-we-can-help")])]), o("td", [t._v("116 123")])]), o("tr", [o("td", [t._v("Israel")]), o("td", [t._v('[Eran] ער"ן')]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.eran.org.il/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.eran.org")])]), o("td", [t._v("1201")])]), o("tr", [o("td", [t._v("Italy")]), o("td", [t._v("Samaritans Onlus")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "http://www.samaritansonlus.org/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.samaritansonlus.org")])]), o("td", [t._v("800 86 00 22")])]), o("tr", [o("td", [t._v("Japan")]), o("td", [t._v("子供こどものSOS")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.mext.go.jp/a_menu/shotou/seitoshidou/06112210.htm",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.mext.go.jp")])]), o("td", [t._v("81-0120-0-78310")])]), o("tr", [o("td", [t._v("Japan")]), o("td", [t._v("まもろうよ こころ")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.mhlw.go.jp/mamorouyokokoro/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.mhlw.go")])]), o("td", [t._v("0570-064-556")])]), o("tr", [o("td", [t._v("Malaysia")]), o("td", [t._v("Befrienders KL")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.befrienders.org.my/centre-in-malaysia",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.befrienders.org")])]), o("td", [t._v("03-76272929")])]), o("tr", [o("td", [t._v("Netherlands")]), o("td", [t._v("113Online")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.113.nl/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.113.nl")])]), o("td", [t._v("0800-0113")])]), o("tr", [o("td", [t._v("New Zealand")]), o("td", [t._v("Lifeline Aotearoa Incorporated")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.lifeline.org.nz/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.lifeline.org")])]), o("td", [t._v("0800 543 354")])]), o("tr", [o("td", [t._v("Norway")]), o("td", [t._v("Mental Helse")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://mentalhelse.no",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("mentalhelse.no")])]), o("td", [t._v("116 123")])]), o("tr", [o("td", [t._v("Philippines")]), o("td", [t._v("Department of Health")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://doh.gov.ph/NCMH-Crisis-Hotline",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("doh.gov.ph/NCMH-Crisis-Hotline")])]), o("td", [t._v("0966-351-4518")])]), o("tr", [o("td", [t._v("Portugal")]), o("td", [t._v("SOS Voz Amiga")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "http://www.sosvozamiga.org",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.sosvozamiga.org")])]), o("td", [t._v(" 213 544 545"), o("br"), t._v("963 524 660"), o("br"), t._v("912 802 669 ")])]), o("tr", [o("td", [t._v("Russia")]), o("td", [t._v("Фонд поддержки детей")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.ya-roditel.ru/parents/helpline/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.ya-roditel.ru")])]), o("td", [t._v("8-800-2000-122")])]), o("tr", [o("td", [t._v("Singapore")]), o("td", [t._v("Samaritans of Singapore")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.sos.org.sg/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.sos.org")])]), o("td", [t._v("1-767")])]), o("tr", [o("td", [t._v("South Africa")]), o("td", [t._v("SA Depression and Anxiety Group")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.sadag.org/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.sadag.org")])]), o("td", [t._v("0800 567 567")])]), o("tr", [o("td", [t._v("South Korea")]), o("td", [t._v("중앙자살예방센터")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "http://www.spckorea.or.kr",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.spckorea.or")])]), o("td", [t._v("1393")])]), o("tr", [o("td", [t._v("Spain")]), o("td", [t._v("Teléfono de la Esperanza")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.telefonodelaesperanza.org/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.telefonodelaesperanza.org")])]), o("td", [t._v("717 003 717")])]), o("tr", [o("td", [t._v("Switzerland")]), o("td", [t._v("Die Dargebotene Hand")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.143.ch/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.143.ch")])]), o("td", [t._v("143")])]), o("tr", [o("td", [t._v("Taiwan")]), o("td", [t._v("国际生命线台湾总会")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "http://www.life1995.org.tw/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.life1995.org")])]), o("td", [t._v("1995")])]), o("tr", [o("td", [t._v("United Kingdom")]), o("td", [t._v("Samaritans")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://www.samaritans.org/how-we-can-help/contact-samaritan",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("www.samaritans.org/how-we-can-help")])]), o("td", [t._v("116 123")])]), o("tr", [o("td", [t._v("United States")]), o("td", [t._v("National Suicide Prevention Lifeline")]), o("td", [o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://suicidepreventionlifeline.org/",
                        rel: "noopener",
                        target: "_blank"
                    }
                }, [t._v("suicidepreventionlifeline.org")])]), o("td", [t._v("988")])])])])
            }
            ]
              , $ = {
                name: "suicide",
                components: {
                    logo: p
                },
                data: function() {
                    return {
                        sot: this.$root.$data
                    }
                }
            }
              , A = $
              , O = (0,
            u.Z)(A, x, E, !1, null, "82a274c2", null)
              , T = O.exports
              , I = function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("div", [t._v(" FREE PALESTINE ")])
            }
              , P = []
              , N = {
                name: "israel",
                components: {},
                data: function() {
                    return {
                        sot: this.$root.$data
                    }
                }
            }
              , j = N
              , z = (0,
            u.Z)(j, I, P, !1, null, "7d116e28", null)
              , H = z.exports
              , D = function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("div", [o("logo"), o("br"), t._v(" Publicnote is a free online notepad. Enter any title to create a new note or to access any existing note. No account is required to access publicnote."), o("br"), o("br"), t._v(" Every note is encrypted with AES-256, using the title as the encryption key. Then an SHA-256 hash of the title is used to index the encrypted note in a database. Since SHA-256 is a one-way function and the process takes place in your browser, the title is never exposed, thus neither the website owner nor the web host (AWS) can view the plaintext contents of any note (unless the title is easily guessed)."), o("br"), o("br"), t._v(" By design, there is no way to recover a note if you forget the title."), o("br"), o("br"), t._v(" Notes stored on publicnote can be public or private depending on the complexity of the title. Simple titles lead to highly visible notes that can be edited by anyone who stumbles upon them, while complex titles lead to private notes that are securely encrypted. The higher the complexity of the title, the stronger the encryption."), o("br"), o("br"), t._v(" Using common words as the title is for entertainment purposes only. You will find a variety of messages left by others. If you find something you find offensive, just delete it."), o("br"), o("br"), t._v(" Do not use publicnote to store sensitive information. By using publicnote you are agreeing to our "), o("div", {
                    staticClass: "link",
                    on: {
                        click: function(e) {
                            t.sot.title = "terms"
                        }
                    }
                }, [t._v("terms")]), t._v("."), o("br"), o("br"), t._v(" Publicnote is "), o("a", {
                    staticClass: "link",
                    attrs: {
                        href: "https://github.com/routman/publicnote",
                        target: "_blank",
                        rel: "noopener"
                    }
                }, [t._v("open source ")]), t._v(" and offered as a free service without any ads. Your donations help fund operating costs and are greatly appreciated. 🙏"), o("br"), o("br"), t._v(" Bitcoin: "), o("p", {
                    staticClass: "addr"
                }, [t._v("bc1q7fqwmtq2vaka8wwpjpnmlehe36qrgfmlw33vh9")]), t._v(" Litecoin: "), o("p", {
                    staticClass: "addr"
                }, [t._v("LYMSJ313xJaUsAmucuYRkVJmGB8Ut9VDz8")]), t._v(" Dogecoin: "), o("p", {
                    staticClass: "addr"
                }, [t._v("DATumCTp1QBG1Gpa3ko6bXPXccnFMFDgYC")]), t._v(" Ethereum: "), o("p", {
                    staticClass: "addr"
                }, [t._v("0x6abD6f3df07c06e4137269D7187661dE37441218")]), t._v(" publicnote © " + t._s((new Date).getFullYear())), o("br"), o("br"), o("div", {
                    staticClass: "link",
                    on: {
                        click: function(e) {
                            t.sot.title = ""
                        }
                    }
                }, [t._v("home")]), o("br"), o("br")], 1)
            }
              , Z = []
              , L = {
                name: "about",
                components: {
                    logo: p
                },
                data: function() {
                    return {
                        sot: this.$root.$data
                    }
                }
            }
              , B = L
              , W = (0,
            u.Z)(B, D, Z, !1, null, "03f1ff3f", null)
              , q = W.exports
              , J = function() {
                var t = this
                  , e = t.$createElement
                  , o = t._self._c || e;
                return o("div", {
                    on: {
                        mouseover: function(e) {
                            t.hover = !0
                        },
                        mouseleave: function(e) {
                            t.hover = !1
                        }
                    }
                }, ["busy" == t.sot.status ? o("div", {
                    attrs: {
                        id: "spinner"
                    }
                }) : t._e(), "ok" == t.sot.status && 0 == t.hover ? o("div", {
                    attrs: {
                        id: "checkmark"
                    }
                }) : t._e(), "ok" == t.sot.status && 1 == t.hover ? o("div", {
                    staticClass: "refresh icon",
                    on: {
                        click: function(e) {
                            return t.sot.get(t.sot.title)
                        }
                    }
                }) : t._e(), "error" == t.sot.status && 0 == t.hover ? o("div", {
                    staticClass: "close icon"
                }) : t._e(), "error" == t.sot.status && 1 == t.hover ? o("div", {
                    staticClass: "refresh icon",
                    on: {
                        click: function(e) {
                            return t.sot.get(t.sot.title)
                        }
                    }
                }) : t._e()])
            }
              , Y = []
              , M = {
                name: "status",
                data: function() {
                    return {
                        sot: this.$root.$data,
                        hover: !1
                    }
                }
            }
              , U = M
              , V = (0,
            u.Z)(U, J, Y, !1, null, "f0ba7ada", null)
              , F = V.exports
              , G = {
                name: "app",
                components: {
                    home: g,
                    terms: S,
                    suicide: T,
                    israel: H,
                    about: q,
                    status: F
                },
                data: function() {
                    return {
                        sot: this.$root.$data,
                        hidden: !0
                    }
                },
                methods: {
                    get: function() {
                        clearTimeout(this.sot.autoload),
                        clearTimeout(this.sot.autosave),
                        "" != this.sot.title && "terms" != this.sot.title && "contact" != this.sot.title && "suicide" != this.sot.title.toLowerCase() && "israel" != this.sot.title.toLowerCase() ? this.sot.get(this.sot.title) : (this.sot.note = "",
                        this.sot.status = "")
                    },
                    save: function() {
                        this.sot.save(this.sot.title, this.sot.note)
                    },
                    show: function() {
                        this.hidden = !1
                    },
                    hide: function() {
                        this.hidden = !0
                    }
                }
            }
              , K = G
              , R = (0,
            u.Z)(K, n, a, !1, null, null, null)
              , X = R.exports
              , Q = o(354);
            r.Z.config.productionTip = !1;
            var tt = null
              , et = null
              , ot = {
                title: "",
                note: "",
                status: "",
                autosave: null,
                autoload: null,
                loading: !1,
                get: function(t) {
                    null != tt && tt.abort(),
                    ot.status = "busy",
                    ot.loading = !0,
                    ot.autoload = setTimeout((function() {
                        var e = Q.SHA256(t).toString()
                          , o = {
                            id: e
                        };
                        tt = new AbortController,
                        et = tt.signal,
                        fetch("https://publicnote.com/api/get", {
                            signal: et,
                            method: "POST",
                            mode: "cors",
                            headers: {
                                "Content-type": "application/json"
                            },
                            body: JSON.stringify(o)
                        }).then((function(t) {
                            return t.json()
                        }
                        )).then((function(e) {
                            var o = JSON.parse(e.body);
                            if (null == o.Item)
                                ot.note = "";
                            else {
                                var r = Q.AES.decrypt(o.Item.ct.toString(), t);
                                ot.note = r.toString(Q.enc.Utf8)
                            }
                            ot.status = "ok",
                            ot.loading = !1,
                            tt = null
                        }
                        )).catch((function(t) {
                            t.toString().search("AbortError") < 0 && (console.log("error: " + t),
                            ot.status = "error")
                        }
                        ))
                    }
                    ), 300)
                },
                save: function(t, e) {
                    ot.status = "busy",
                    clearTimeout(ot.autosave),
                    ot.autosave = setTimeout((function() {
                        var o = Q.SHA256(t).toString()
                          , r = Q.AES.encrypt(e, t).toString()
                          , n = {
                            id: o,
                            ct: r
                        };
                        fetch("https://publicnote.com/api/save", {
                            method: "POST",
                            mode: "cors",
                            headers: {
                                "Content-type": "application/json"
                            },
                            body: JSON.stringify(n)
                        }).then((function(t) {
                            return t.json()
                        }
                        )).then((function(t) {
                            "successfully saved" == t.body ? ot.status = "ok" : ot.status = "error"
                        }
                        )).catch((function(t) {
                            console.log("error saving: " + t),
                            ot.status = "error"
                        }
                        )),
                        ot.autosave = null
                    }
                    ), 2e3)
                }
            };
            new r.Z({
                render: function(t) {
                    return t(X)
                },
                data: ot
            }).$mount("#app")
        }
    }
      , e = {};
    function o(r) {
        var n = e[r];
        if (void 0 !== n)
            return n.exports;
        var a = e[r] = {
            exports: {}
        };
        return t[r].call(a.exports, a, a.exports, o),
        a.exports
    }
    o.m = t,
    function() {
        var t = [];
        o.O = function(e, r, n, a) {
            if (!r) {
                var s = 1 / 0;
                for (d = 0; d < t.length; d++) {
                    r = t[d][0],
                    n = t[d][1],
                    a = t[d][2];
                    for (var i = !0, l = 0; l < r.length; l++)
                        (!1 & a || s >= a) && Object.keys(o.O).every((function(t) {
                            return o.O[t](r[l])
                        }
                        )) ? r.splice(l--, 1) : (i = !1,
                        a < s && (s = a));
                    if (i) {
                        t.splice(d--, 1);
                        var c = n();
                        void 0 !== c && (e = c)
                    }
                }
                return e
            }
            a = a || 0;
            for (var d = t.length; d > 0 && t[d - 1][2] > a; d--)
                t[d] = t[d - 1];
            t[d] = [r, n, a]
        }
    }(),
    function() {
        o.d = function(t, e) {
            for (var r in e)
                o.o(e, r) && !o.o(t, r) && Object.defineProperty(t, r, {
                    enumerable: !0,
                    get: e[r]
                })
        }
    }(),
    function() {
        o.g = function() {
            if ("object" === typeof globalThis)
                return globalThis;
            try {
                return this || new Function("return this")()
            } catch (t) {
                if ("object" === typeof window)
                    return window
            }
        }()
    }(),
    function() {
        o.o = function(t, e) {
            return Object.prototype.hasOwnProperty.call(t, e)
        }
    }(),
    function() {
        var t = {
            143: 0
        };
        o.O.j = function(e) {
            return 0 === t[e]
        }
        ;
        var e = function(e, r) {
            var n, a, s = r[0], i = r[1], l = r[2], c = 0;
            if (s.some((function(e) {
                return 0 !== t[e]
            }
            ))) {
                for (n in i)
                    o.o(i, n) && (o.m[n] = i[n]);
                if (l)
                    var d = l(o)
            }
            for (e && e(r); c < s.length; c++)
                a = s[c],
                o.o(t, a) && t[a] && t[a][0](),
                t[a] = 0;
            return o.O(d)
        }
          , r = self["webpackChunkpublicnote"] = self["webpackChunkpublicnote"] || [];
        r.forEach(e.bind(null, 0)),
        r.push = e.bind(null, r.push.bind(r))
    }();
    var r = o.O(void 0, [998], (function() {
        return o(830)
    }
    ));
    r = o.O(r)
}
)();
//# sourceMappingURL=app.710d8a38.js.map

```
You cannot find all the references to Israel from the source code above in the repo.

Given the above, the author's claims of privacy and security cannot be trusted and you should use this site with extreme caution ⚠️

**Comment by routman** [comment]
There you go.  I updated the repo just for you, pal.

Disingenuous? What about the notes for "about" and "suicide"?  Does the fact that I made those notes uneditable make me a dishonest and disingenuous person? What an incredibly absurd and stupid thing to say.

I think what's really going on here is that you are personally offended by the fact that I hard coded "FREE PALESTINE" into the note for Israel.  Israel is committing genocide as we speak, therefore I consider this a non issue.  Case closed. 👍
